robocop/rules.py

```diff
@@ -190,9 +190,15 @@ def supported_in_rf_version(version: str) -> bool:
     def __str__(self):
         return (
             f"Rule - {self.rule_id} [{self.config['severity'].value}]: {self.name}: {self.message_for_docs} "
-            f'({"enabled" if self.enabled else "disabled"})'
+            f'({self.get_enabled_status_desc()})'
         )
 
+    def get_enabled_status_desc(self):
+        s = "enabled" if self.enabled else "disabled"
+        if not self.enabled and self.supported_version != "All":
+            s += f" - enabled for RF version {self.supported_version}"
```

Comment:
It looks strange and may be confusing that 'disabled' and 'enabled' words are next to each other.
I suggest changing it to:
```suggestion
            s += f" - supported only for RF version {self.supported_version}"
```